linalg/ode.hpp

```diff
@@ -70,6 +70,9 @@ class ODESolver
          between the two Step() calls. */
    virtual void Step(Vector &x, double &t, double &dt) = 0;
 
+   //  virtual void Step(Vector &x, Vector &dxdt, double &t, double &dt)
```

Comment:
should remove this commented out function

Reply:
Done.